methods.py

```diff
@@ -254,6 +250,11 @@ def get_cmdline_bool(option, default):
         return default
 
 
+############################################################
+# MODULE LOADING
+############################################################
```

Comment:
Forgot to clarify that this delimiter is a followup to https://github.com/godotengine/godot/pull/98653